File: app_manager/core/src/main/java/app/documents/core/network/manager/models/request/RequestRenameFile.java
package app.documents.core.network.manager.models.request;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class RequestRenameFile {

    @SerializedName("title")
    @Expose
    private String title;
    @SerializedName("lastVersion")
    @Expose
    private int lastVersion;

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public int getLastVersion() {
        return lastVersion;
    }

    public void setLastVersion(int lastVersion) {
        this.lastVersion = lastVersion;
    }

}

File: app_manager/core/src/main/java/app/documents/core/network/manager/models/request/RequestRenameFile.kt
package app.documents.core.network.manager.models.request

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class RequestRenameFile(
    @SerializedName("title")
    @Expose
    var title: String? = null,

    @SerializedName("lastVersion")
    @Expose
    var lastVersion: Int = 0
)